mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -3919,6 +3919,15 @@ struct Conv2DRewritePattern : public OpRewritePattern<T> {
                            b.getArrayAttr({b.getStringAttr("hi"),
                                            b.getStringAttr("wi")}))};
         auto isInputHipBoundCheck = [&]() {
+          // FIXME:  wTildaSlice > wo will let padding kernel fail,
+          // we need to fix the padding kernel in next step
+          // but this logic(wTildaSlice > wo) is correct
```

Comment:
This comment can’t be understood easily. Please try rephrase it.

it seems you are stating something is wrong but the logic is correct?